.changeset/sharp-dolls-confess.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
documentation changes don't require a changeset

Reply:
it was the test changes that I thought might need a lil patch, but no worries, I'll remove it